USAGE.md

```diff
@@ -30,6 +30,7 @@ See the [Hue Emulation](https://www.openhab.org/addons/integrations/hueemulation
   Some examples of tagged items are:
   
   ```java
+  Switch Item_Name(Required) "Item Label"(Required) <Icon Name>(optional) [Tag](Required) {channel="..."}(optional)
```

Comment:
So the example here is meant to be a working item file example, the line you added here is not a valid syntax, so I would prefer not to add that here.